application/classes/Ushahidi/Console/Notification/Collection.php

```diff
@@ -91,7 +98,7 @@ protected function executeQueue(InputInterface $input, OutputInterface $output)
 		];
 	}
 
-	private function _generate_messages($notification)
+	private function generateMessages($notification)
```

Comment:
This should move into a Usecase eventually. **not now** but add a tech debt task to track it